client/src/components/CreateBoardModal/styles.module.scss

```diff
@@ -0,0 +1,11 @@
+@import 'styles/mixins.scss';
```

Comment:
if you don't have any other styles you don't need skoped styles, cause those classNames appear in styles/index.scss

Reply:
@Alesyaa103 

Fixed in https://github.com/BinaryStudioAcademy/bsa-2020-TaskyHusky/pull/301/commits/3f3ece94734e375ff5dcdcb0baa34611d386ae82